### PR TITLE
Cleanup netty-http logs

### PR DESCRIPTION
**Pull request by deepakwadhwani**
CDAP-9321: 
Cleanup logs from netty-http

### CODE REVIEW

File: src/main/java/co/cask/http/NettyHttpService.java [hunk 1]
```diff
@@ -285,9 +285,14 @@ protected void shutDown() throws Exception {
         ExecutorUtil.terminate(executionHandler.getExecutor());
       }
     }
-    LOG.info("Stopped {} http service on address {}", serviceName, bindAddress);
+    LOG.debug("Stopped HTTP Service {} on address {}", serviceName, bindAddress);
   }
 
+  public String getSimpleServiceName(String serviceName) {
+    return serviceName.substring(serviceName.lastIndexOf(".") + 1);
```
Comment (anwar6953) [inline src/main/java/co/cask/http/NettyHttpService.java]: can we just change where the `serviceName` is passed in from, to do `Class.getSimpleName()`?

Comment (deepakwadhwani) [reply src/main/java/co/cask/http/NettyHttpService.java]: @anwar6953 Might be helpful to leave the qualified name in the debug log?

Comment (chtyim) [reply src/main/java/co/cask/http/NettyHttpService.java]: What is this for? We should change the caller to always pass in a service name.

Comment (deepakwadhwani) [reply src/main/java/co/cask/http/NettyHttpService.java]: @chtyim debug logs can carry the qualified service name this way, in case its ever required.

Comment (awholegunch) [reply src/main/java/co/cask/http/NettyHttpService.java]: Seems an odd place to put knowledge of the structure of the simple service name. Should that not be closer to the instantiation?

**Comment by awholegunch** [comment]
The messages are OK. Other issues need to be resolved.

File: src/main/java/co/cask/http/NettyHttpService.java (hunk 1 above)
Comment (chtyim) [reply src/main/java/co/cask/http/NettyHttpService.java]: What I mean is we shouldn't use simple class name as the service name.

Comment (deepakwadhwani) [reply src/main/java/co/cask/http/NettyHttpService.java]: ok, I removed this change here and I pass the Program Name for service name for Spark programs.

**Comment by chtyim** [comment]
LGTM

**Pull request merged**